vignettes/package-use-cases.Rmd

```diff
@@ -0,0 +1,46 @@
+---
+title: Package use cases
+description: "A place to document how epinowcast has been used"
+author: 
+output:
+  bookdown::html_vignette2:
+    fig_caption: yes
+    code_folding: show
+pkgdown:
+  as_is: true
+
+csl: https://raw.githubusercontent.com/citation-style-language/styles/master/apa-numeric-superscript-brackets.csl
+vignette: >
+  %\VignetteIndexEntry{Package use cases}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  eval = FALSE
+)
+```
+
+The goal of this vignette is to provide a place to document how `{epinowcast}` has been applied in real-world settings. 
+To start, we will document a few known use cases, many of which the authors are directly involved in. 
+We hope this can both inspire future users to apply the methods to similar use cases of their own, and those currently or previously using `{epinowcast}` to document it here. This should also motivate and help prioritize future development.
```

Comment:
How do they document it here (open a PR I would assume)

Reply:
Yes can add this. My thought was just that any time package developers become aware of someone using the package they could send a link to this vignette and suggest opening a PR to add to the table